components/AllocatedWorkers/AllocatedWorkersTable.jsx

```diff
@@ -1,4 +1,8 @@
+import Modal from 'components/Modal/Modal';
 import PropTypes from 'prop-types';
+import Button from '../Form/Button/Button';
+import { useState } from 'react';
```

Comment:
It's not changing much, but by convention, the external dependencies (i.e. `react`, `prop-types`) should be at the top.